### DISCUSSION TITLE
docker 安装 grafana mivus2.1 没有展示数据, 求一份docker下安装流程

### DISCUSSION THREAD

**Discussion by zhoushewei** [milvus-io/milvus#19110] [Q&A and General discussion]
milvus V2.11  docker 安装
grafana,prometheus  docker安装

milvus-dashboard.json 使用官网下载的配置

prometheus Targets ：
![image](https://user-images.githubusercontent.com/42832745/189076899-290142be-ff5c-4d3c-8c11-fe90369598ae.png)

grafana：
![image](https://user-images.githubusercontent.com/42832745/189077111-e8b63f55-7773-429d-8b4a-d4c35d956ea3.png)

milvus.yaml
![image](https://user-images.githubusercontent.com/42832745/189077443-058915a7-f668-4c6b-89c0-8e08071ee242.png)

docker-compose.yml：
![image](https://user-images.githubusercontent.com/42832745/189077599-ed9056c9-fee1-4bb9-8f48-abeaf53e9bb8.png)

**Comment by yhmo** [comment]
用docker部署基本上就没法用，必须得用k8s来部署才行，因为里面用到了很多k8s提供的功能。
我们文档里的步骤也是基于k8s的：https://milvus.io/docs/v2.1.x/monitor.md
